lib/rules/attribute-indentation.js

```diff
@@ -38,6 +38,17 @@ function canApplyRule(node, type, config) {
 }
 
 module.exports = class AttributeSpacing extends Rule {
+  static get meta() {
+    return {
+      description: 'requires params & attributes to be indented',
+      category: 'Stylistic Issues', // 'Deprecated Rules', 'Possible Error', 'Best Practices',
+      recommended: false,
+      url:
+        'https://github.com/ember-template-lint/ember-template-lint/blog/master/docs/rules/attribute-indentation.md',
+      fixable: false,
```

Comment:
maybe we can derive this one automatically based on the existance of a fixer method on the rule or something like that? I haven't looked into the fixer implementation yet.

Reply:
The fixer implementation will not be "across the board" for each rule either, each logged failure would be independently fixable (meaning if we tried to detect if fixing is possible, we'd first have to actually lint a file!).

Since, this is a static getter on the class (with the intention to be able to easily use it for scripting things like building the README) I think we should keep a flag here to indicate that it _may_ be fixable.